appinventor/appengine/src/com/google/appinventor/client/actions/ShowShortcutsAction.java

```diff
@@ -0,0 +1,60 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2009-2011 Google, All Rights reserved
+// Copyright 2011-2023 MIT, All rights reserved
+// Released under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package com.google.appinventor.client.actions;
+
+import static com.google.appinventor.client.Ode.MESSAGES;
+
+import com.google.gwt.dom.client.NativeEvent;
+import com.google.gwt.event.dom.client.KeyCodes;
+import com.google.gwt.user.client.Command;
+import com.google.gwt.user.client.Event;
+import com.google.gwt.user.client.ui.DialogBox;
+import com.google.gwt.user.client.ui.HTML;
+import com.google.gwt.user.client.ui.VerticalPanel;
+
+public class ShowShortcutsAction implements Command {
+
+    private DialogBox db;
+
+    public ShowShortcutsAction() {
+        db = new DialogBox(true, false);
+        db.setText("Keyboard Shortcuts");
+        db.setStyleName("ode-DialogBox");
+        db.setHeight("200px");
+        db.setWidth("400px");
+        db.setGlassEnabled(true);
+        db.setAnimationEnabled(true);
+
+        shortcutKeyHandler();
+    }
+
+    @Override
+    public void execute() {
+        VerticalPanel DialogBoxContents = new VerticalPanel();
```

Comment:
This dialog doesn't have much to put in a UIBinder template, but in order to match the convention in the rest of the app, it should have an OK button at the bottom to close it

Reply:
This is a rethinking of the earlier comment about using UIBinder for this dialog. We're trying to convert all our dialogs to the UIBinder templates, but this literally just an information box with no functionality, more like the help popups for components and properties. I think it should have an OK to close, since everything else access from the toolbar has that behavior, but it seems OK to skip a template.